### DISCUSSION TITLE
Creating Payment Method or Card Token Results in Stripe Exception: Generally Unsafe

### DISCUSSION THREAD

**Discussion by michaeljajou** [flutter-stripe/flutter_stripe#1542] [Q&A]
**Describe the bug**
Creating a token or payment method fails with a Stripe Exception:

StripeException(error: LocalizedErrorMessage(code: FailureCode.Failed, localizedMessage: There was an unexpected error -- try again in a few seconds, message: Sending credit card numbers directly to the Stripe API is generally unsafe. To continue processing use Stripe.js, the Stripe mobile bindings, or Stripe Elements. For more information, see https://dashboard.stripe.com/account/integration/settings. If you are qualified to handle card data directly, see https://support.stripe.com/questions/enabling-access-to-raw-card-data-apis., stripeErrorCode: , declineCode: null, type: invalid_request_error))

```
stripe.Stripe.publishableKey = Flavor.stripeKey;
final params = stripe.PaymentMethodParams.card(
  paymentMethodData: stripe.PaymentMethodData(
    billingDetails: stripe.BillingDetails(),
  ),
);
final method = await _stripe.createPaymentMethod(params: params);
print(method.id);
return method.id;
```

Or by Creating a Token
```
stripe.Stripe.publishableKey = Flavor.stripeKey;
const request = stripe.CreateTokenParams.card(
  params: stripe.CardTokenParams(),
);
final token = await _stripe.createToken(request);
print(token);
return token.id;
```

**Comment by remonh87** [comment]
I would recommend checking out: https://github.com/flutter-stripe/flutter_stripe/blob/main/example/lib/screens/others/legacy_token_card_screen.dart
